tests/models/test_tpu.py

```diff
@@ -319,9 +319,9 @@ def test_broadcast(rank):
 def test_tpu_choice(tmpdir, tpu_cores, expected_tpu_id, error_expected):
     if error_expected:
         with pytest.raises(MisconfigurationException, match=r".*tpu_cores` can only be 1, 8 or [<1-8>]*"):
-            Trainer(default_root_dir=tmpdir, accelerator="tpu", devices=tpu_cores)
+            Trainer(default_root_dir=tmpdir, tpu_cores=tpu_cores)
```

Comment:
qq: this isn't completely compatible yet?

Reply:
Updated the test!